api/controllers/CampaignBotController.js

```diff
@@ -452,37 +375,31 @@ CampaignBotController.prototype.postReportback = function(req, res) {
  * @param {object} res - Express response
  */
 CampaignBotController.prototype.supportsMMS = function(req, res) {
-  var self = this;
+  this.debug(req, 'supportsMMS');
 
-  if (self.user.supports_mms === true) {
+  if (this.user.supports_mms === true) {
     return true;
   }
+
   // @todo DRY this logic.
   // @see loadSignup()
-  var incomingCommand = helpers.getFirstWord(req.incoming_message);
+  const incomingCommand = helpers.getFirstWord(req.incoming_message);
   if (incomingCommand && incomingCommand.toUpperCase() === CMD_REPORTBACK) {
-    self.sendMessage(req, res, self.bot.msg_ask_supports_mms);
+    this.sendMessage(req, res, this.bot.msg_ask_supports_mms);
     return false;
   }
   
   if (!helpers.isYesResponse(req.incoming_message)) {
-    self.sendMessage(req, res, self.bot.msg_no_supports_mms);
+    this.sendMessage(req, res, this.bot.msg_no_supports_mms);
     return false;
   }
 
-  self.user.supports_mms = true;
-
-  self.user.save(function (err) {
-
-    if (err) {
-      return self.handleError(err);
-    }
-
-    self.startReportbackSubmission(req, res);
-    return true;
-
-  });
-
+  this.user.supports_mms = true;
+  return this.user
```

Comment:
I think this helps make the argument to keep `this.user` saved on our `CampaignBotController` and not `req` (refs https://github.com/DoSomething/gambit/pull/628/files#r80141327)